openassessment/xblock/test/test_grade_explanation.py

```diff
@@ -0,0 +1,263 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for grade explanation in Open Response Assessment XBlock.
+"""
+from __future__ import absolute_import
+
+import json
+
+from django.conf import settings
+from ddt import ddt, data
+
+from .base import (PEER_ASSESSMENTS, SELF_ASSESSMENT, STAFF_GOOD_ASSESSMENT,
+                   SubmitAssessmentsMixin, XBlockHandlerTestCase, scenario)
+
+
+@ddt
+class TestGradeExplanation(XBlockHandlerTestCase, SubmitAssessmentsMixin):
+    """
+    Tests for grade explanation in Open Response Assessment XBlock.
+    """
+
+    second_sentences_options = {
+        "self": "The grade for this problem is determined by your Self Assessment.",
+        "staff": "The grade for this problem is determined by your Staff Grade.",
+        "peer": "The grade for this problem is determined by the median score of your Peer Assessments."
+    }
+
+    assessment_score_priority = (
+        ('self', 'peer'),
+        ('peer', 'self')
+    )
+
+    @scenario('data/grade_scenario_self_only.xml', user_id='Greggs')
+    @data(*assessment_score_priority)
+    def test_render_grade_explanation_self_only(self, xblock, assessment_score_priority):
+        settings.ORA2_ASSESSMENT_SCORE_PRIORITY = assessment_score_priority
+        self.create_submission_and_assessments(
+            xblock, self.SUBMISSION, [], [], SELF_ASSESSMENT,
+            waiting_for_peer=True
+        )
+        resp = self.request(xblock, 'render_grade', json.dumps(dict()))
+
+        self.assertIn(self.second_sentences_options["self"], resp.decode('utf-8'))
+
+    @scenario('data/grade_scenario_staff_only.xml', user_id='Bernard')
+    @data(*assessment_score_priority)
+    def test_render_explanation_grade_staff_only(self, xblock, assessment_score_priority):
```

Comment:
Why test with 2 different priorities here if the final outcome doesn't matter for staff graded problems? 
Can you remove `@data(*assessment_score_priority)` and `assessment_score_priority` for staff related tests?

Reply:
Yes, that what we wanted to test that **"No matter what the priority is the outcome would be based on staff grade"**
If you still want me to remove it, I would.